### ISSUE TITLE
Use Whoops with Mini3

### ISSUE THREAD

**Issue by HawkonDK** [panique/mini3#35]
How to implement Whoops with Mini3 ?

**Comment by mhco** [comment]
It certainly is possible, the only question is how much time would it take, and at what point would it be better for you and your project(s) to just use Laravel, instead of Mini3?

https://github.com/XiaoLer/blade seems to have extracted Blade from Laravel.

But it would require a good amount of work to get it integrated properly (as it's a complex templating engine, and Mini3 is an ultra-slim framework). It's probably not a project for the timid, honestly.

**Comment by panique** [comment]
Yeah it's possible but as MINI tries to be as simple as possible (without any external plugins etc) it's probably the best to use pure and raw PHP instead of blade!
